src/styles/index.scss

```diff
@@ -1,3 +1,102 @@
+@import url("./star.scss");
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  line-height: 18px;
+  color: #060b35;
+
+  --color-blue: #00acdc;
+}
+
+h2 {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
```

Comment:
do not forget about css properties order
http://codeguide.academy/html-css.html#css-order